scripts/go-release-automation.sh

```diff
@@ -71,7 +72,8 @@ run_release_script() {
   # Replacement directives are removed to get package from go pkg instead of local copy
   # We need to copy `go.mod` otherwise without it this script will not work in the first release because all the go tools used in this script works only when there is a `go.mod` in the directory. Removing replacement directives helps in automating copying of `go.mod`.
   echo "Removing all replace directives from go.mod..."
-  go mod edit -json | jq -r '.Replace[].Old.Path' | xargs -n1 go mod edit -dropreplace
+  # This expects the replaced package does not have a version
```

Comment:
I wanted to make this work for all the cases but I figured I am playing a lot with the string and it will just cause more issue. So, I decided to put a if condition and make it fail when replacement character is found